Source/WebKit/NetworkProcess/NetworkProcess.cpp

```diff
@@ -459,7 +459,7 @@ void NetworkProcess::clearCachedCredentials(PAL::SessionID sessionID)
     if (auto* session = networkSession(sessionID)) {
         if (auto* storageSession = session->networkStorageSession())
             storageSession->credentialStorage().clearCredentials();
-        session->clearCredentials();
+        session->clearCredentials(WallTime { });
```

Comment:
This is a change in behavior, though probably a good one.

Reply:
oh right, the old function is a no-op. Will update commit message.